### ISSUE TITLE
thread safe

### ISSUE THREAD

**Issue by zyx1996** [ARM-software/optimized-routines#72]
Hi，Is it safe to use the same function simultaneously in multiple threads？

**Comment by Wilco1** [comment]
Yes. This is generally true for all math functions and almost all string functions in C/C++ (eg. strtok an exception but not supported here).
